.github/release-drafter.yml

```diff
@@ -23,6 +23,7 @@ categories:
       - 'automation'
       - 'documentation'
       - 'dependencies'
+      - 'revert'
```

Comment:
Adding just in case or do you see an immediate need for this?
Also, can you create this label in the repo?

Reply:
I didn't think it aligned with any of the pre-existing labels.  Do you agree or is there one you think it may align with?

I can create the label in the repo if we want to keep it.

Reply:
let's add it. I see now how the pr titles are getting matched to a label.